### ISSUE TITLE
AvoidUsingObjectMapperAsALocalVariable is allowed in methods with annotation org.springframework.context.annotation.Bean

### ISSUE THREAD

**Issue by dgroup** [dgroup/arch4u-pmd#144]
### Summary
| Key            |  Value |
|:---------------|:------|
| Rule           | AvoidUsingObjectMapperAsALocalVariable |
| arch4u version       | 0.2.2 |

### Your rule definition
```xml
default rule configuration from `arc4u-pmd:0.2.2`
```

### Describe the defect or false/positive rule violation
```java
    @Bean
    ObjectMapper objectMapper() {
        ObjectMapper objectMapper = new ObjectMapper();
        objectMapper.setSerializationInclusion(JsonInclude.Include.NON_NULL);
        return objectMapper;
    }
```
